app/common/lib/ledgerExportUtil.js

```diff
@@ -234,13 +234,13 @@ module.exports.getTransactionCSVRows = (transactions, viewingIds, addTotalRow, s
 
   const currency = (publishers.length ? txContribData[publishers[0]].contribution.currency : 'USD')
 
-  const headerRow = ['Publisher', 'Votes', 'Fraction', 'BTC', currency].join(',')
+  const headerRow = ['Publisher', 'Votes', 'Fraction', 'BAT', currency].join(',')
```

Comment:
looking at this, this brings up an interesting question... what do previous transactions look like?

ex: we switch over to BAT. Can I still view the reports for the Bitcoin payments I've made? Do we store (in the state) which digital currency was used to settle each transaction?

Reply:
nice catch I forgot to implement this part, which @mrose17 told me about it. @bsclifton can you please crate issue for it, so that I don't forget :smiley: (add release-blocking to it)

Reply:
will be fixed with #11296